docs/api/rest_api/rest_api_reference/input/ez-orders.raml

```diff
@@ -79,30 +79,52 @@ displayName: Orders
         /shipments:
             get:
                 displayName: Load shipments
-                description: Loads a list of shipments.
+                description: Loads a order's list of shipments.
                 headers:
                     Accept:
                         description: If set, the shipments are returned in XML or JSON format.
                         example:  |
                             application/vnd.ibexa.api.ShipmentRefList+xml
-                    Content-Type:
-                        description: The shipments schema encoded in XML or JSON format.
-                        example:  |
-                            application/vnd.ibexa.api.ShipmentRefList+xml
                 responses:
                     200:
                         description: OK - loads the shipments.
                         body:
-                            application/vnd.ibexa.api.Shipment+xml:
-                                type: Shipment
+                            application/vnd.ibexa.api.ShipmentRefList+xml:
+                                type: ShipmentRefList
                                 example: !include examples/shipments/GET/ShipmentRefList.xml.example
-                            application/vnd.ibexa.api.Shipment+json:
-                                type: Shipment
+                            application/vnd.ibexa.api.ShipmentRefList+json:
+                                type: ShipmentRefList
                                 example: !include examples/shipments/GET/ShipmentRefList.json.example
                     401:
-                        description: Error - the user has no permission to read shipment lists.
+                        description: Error - the user has no permission to read order's shipment lists.
                     404:
-                        description: Error - the shipment does not exist.
+                        description: Error - the order does not exist.
+            post:
+                displayName: Create shipment
+                description: Creates and adds a shipment to an order
+                headers:
+                    Accept:
+                        description: If set, the shipments are returned in XML or JSON format.
+                        example: |
+                            application/vnd.ibexa.api.Shipment+xml
+                            application/vnd.ibexa.api.Shipment+json
+                    Content-Type:
+                        description: The shipments schema encoded in XML or JSON format.
+                        example: |
+                            application/vnd.ibexa.api.ShipmentCreate+xml
+                            application/vnd.ibexa.api.ShipmentCreate+json
+                body:
+                    application/vnd.ibexa.api.ShipmentCreate+xml:
+                        type: ShipmentCreate
```

Comment:
We probably need XML payload example as well, as we provide for the other endpoints.

Reply:
Added in ef48c4ef4c887959331499dc4263621de072fe85 thanks to @alongosz